src/patternfly/patternfly.scss

```
@@ -4,3 +4,30 @@
+    background-image: url("./assets/images/glass-brand-light.jpg");
+    // background-image: url("./assets/images/compass--wallpaper-light.jpg");
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: center;
+    background-size: cover;
+  }
+
+  &:where(.pf-v6-theme-dark) {
+    .pf-v6-c-page, .pf-v6-c-login, body {
+      background-image: url("./assets/images/glass-brand-dark.jpg");
+      // background-image: url("./assets/images/compass--wallpaper-dark.jpg");
```

Stylelint (17.4.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)